test/services/GovernmentSpendServiceSpec.scala

```diff
@@ -96,4 +100,32 @@ class GovernmentSpendServiceSpec extends UnitSpec with GuiceOneAppPerSuite with
       )
     }
   }
+
+  "GovernmentSpendService getGovernmentSpendDataV2" should {
+
+    "return a government spend map" in {
+
+      val expectedBody = Seq(("Environment", 5.5))
+
+      when(mockMiddleConnector.connectToGovernmentSpend(meq(taxYear), meq(testNino))(any())) thenReturn Future
+        .successful(HttpResponse(OK, Json.parse("""{"Environment":5.5}"""), Map("" -> Seq(""))))
+
+      val result = sut.getGovernmentSpendDataV2(taxYear, Some(testNino)).futureValue
+
+      result shouldBe expectedBody
+    }
+
+    "sort data by percentage" in {
+
```

Comment:
Might be wrong here but don't we have a category that has equal percentage that needs to be swapped around?

Reply:
This is just to test that the data is sorted from high to low by percentage during that function